.github/workflows/release.yml

```
@@ -12,22 +14,20 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4.2.2
-      - uses: actions/setup-python@v5.4.0
         with:
-          python-version: ">=3.10"
-      - name: Get version
-        run: |
-          echo "version=$(date +'%Y.%m.%d.%H.%M')" >> "$GITHUB_ENV"
-      - name: Tag and release
-        env:
-          GH_TOKEN: ${{ github.token }}
+          fetch-depth: 0
+          fetch-tags: true
+
+      - uses: actions/setup-python@v5.3.0
+        with:
+          python-version: 3.12
+
+      - name: Install build
         run: |
-            git tag $version
-            git push origin $version
-            pip install poetry
-            poetry self add poetry-dynamic-versioning[plugin]
-            poetry install --sync
-            poetry build
-            gh release create --generate-notes $version ./dist/*
+          pip install poetry
+          poetry self add poetry-dynamic-versioning[plugin]
+          poetry install --sync
```

```suggestion
          poetry sync
```
I guess Poetry 2.0.1 is installed now.